deploy/helm/charts/templates/deployment.yaml

```diff
@@ -83,6 +83,18 @@ spec:
             # leader election is enabled.
             - name: LEADER_ELECTION_ENABLED
               value: "{{ .Values.nfsProvisioner.enableLeaderElection }}"
+            # OPENEBS_IO_NFS_SERVER_NODEAFFINITY defines the node affinity rules to place NFS Server
+            # instance. It accepts affinity rules in multiple ways:
+            # - If NFS Server needs to be placed on storage nodes as well as only in
+            #   zone-1 & zone-2 then value can be:
+            #   value:  "kubernetes.io/zone:[zone-1,zone-2],kubernetes.io/storage-node".
+            # - If NFS Server needs to be placed only on storage nodes & nfs nodes then
+            #   value can be:
+            #   value:  "kubernetes.io/storage-node,kubernetes-nfs-node"
+            {{- if .Values.nfsProvisioner.nfsServerNodeAffinity }}
```

Comment:
Need to update the values.yaml file with commented value.

Reply:
updated values.yaml with commented lines